hdc10102GradedAssessment/script-hdc10102Task03.txt

```diff
@@ -0,0 +1 @@
+https://
```

Comment:
aren't we missing the hashscan url here for account 0.0.10?

Reply:
This is intentional, this is the version that is incomplete, and the student is to replace it with the full URL. (so this serves as more of a hint)